src/main/java/com/acm/casemanagement/controller/UserController.java

```diff
@@ -87,7 +87,7 @@ public ResponseEntity<Void> deleteUser(@PathVariable Long id) {
         Optional<User> user = userService.getUserById(id);
         if (user.isPresent()) {
             userService.deleteUserById(id);
-            return ResponseEntity.noContent().build();
+            return ResponseEntity.noContent().build(); //response code 204
         } else {
             return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
```

Comment:
The service class handled  the User not found scenario properly and no need to add it here.